crates/ty_python_semantic/src/types/display.rs

```diff
@@ -324,6 +326,7 @@ impl Display for DisplayGenericContext<'_> {
             if idx > 0 {
                 f.write_str(", ")?;
             }
+            write!(f, "{}", var.variance(self.db))?;
```

Comment:
@AlexWaygood will change this to
```suggestion
            var.variance(self.db).fmt(f)?;
```

Reply:
This is moot since I reverted the rendering changes